wagtail/admin/wagtail_hooks.py

```diff
@@ -874,6 +874,11 @@ def is_shown(self, request):
         )
 
 
+class ScheduledPagesMenuItem(MenuItem):
+    def is_shown(self, request):
```

Comment:
We should have a permission check here (and in the view itself) to ensure that this report is hidden from users who have no add/change/publish permissions over pages at all - similar to the ones added in #11205.

Reply:
Good point, that's been added now.